R/get_match_report.R

```diff
@@ -22,7 +22,7 @@
     game <- each_game_page %>% rvest::html_nodes("h1") %>% rvest::html_text()
 
     # tryCatch( {League <- each_game_page %>% rvest::html_nodes("h1+ div a") %>% rvest::html_text()}, error = function(e) {League <- NA})
-    tryCatch( {League <- each_game_page %>% rvest::html_nodes("h1+ div a") %>% rvest::html_text()}, error = function(e) {League <- NA})
+    tryCatch( {League <- each_game_page %>% rvest::html_nodes("h1+ div a:nth-child(1)") %>% rvest::html_text()}, error = function(e) {League <- NA})
```

Comment:
Was causing issues when there were multiple legs of cup ties with downstream functions that called this internal function